packages/upload-api/src/blob/replicate.js

```diff
@@ -171,13 +185,19 @@ export const blobReplicateProvider = (context) => {
                 site: nb.site,
                 cause: invocation.cid,
               },
+              facts: allocFacts,
```

Comment:
I thought attaching the blocks was enough. What would happen if they are not added as facts? Would the blocks be stripped from the invocation if they are not referenced?